Public/Src/Utilities/Utilities/Tracing/EventId.cs

```diff
@@ -570,6 +570,7 @@ public enum EventId
         ConfigUnsafeAllowMissingOutput = 928,
         ConfigIgnoreValidateExistingFileAccessesForOutputs = 929,
         ConfigUnsafeIgnoreUndeclaredAccessesUnderSharedOpaques = 930,
+        ConfigUnsafeOptimizedAstConversion = 931,

```

Comment:
Please don't ass new ones to Utilities.Tracing. Please add to the Tracing\log.cs file in the project that raises the event... In this case `Public/src/Engine/Dll/Tracing/log.cs`

Reply:
We should do a clean sweep in a separate PR.